packages/broker/test/integration/multi-account-brokers.test.ts

```diff
@@ -0,0 +1,800 @@
+                key: 4
+            },
+        ])
+    })
+
+    it('happy-path: resend from request via websocket', async () => {
+        await Promise.all([
+            client1.subscribe({
+                stream: freshStreamId
+            }, () => {}),
+            client2.subscribe({
+                stream: freshStreamId
+            }, () => {}),
+            client3.subscribe({
+                stream: freshStreamId
+            }, () => {}),
+        ])
+
+        await client1.publish(freshStreamId, {
+            key: 1
+        })
+        await wait(50)
+        const timeAfterFirstMessagePublished = Date.now()
+
+        await client1.publish(freshStreamId, {
+            key: 2
+        })
+        await wait(50)
+        await client1.publish(freshStreamId, {
+            key: 3
+        })
+        await wait(50)
+        await client1.publish(freshStreamId, {
+            key: 4
+        })
+
+        await wait(1500) // wait for propagation
+
+        const client1Messages: Todo[] = []
+        const client2Messages: Todo[] = []
+        const client3Messages: Todo[] = []
+        await Promise.all([
+            client1.resend({
+                stream: freshStreamId,
+                resend: {
+                    from: {
+                        timestamp: timeAfterFirstMessagePublished,
+                    }
+                }
+            }, (message) => {
+                client1Messages.push(message)
+            }),
+            client2.resend({
+                stream: freshStreamId,
+                resend: {
+                    from: {
+                        timestamp: timeAfterFirstMessagePublished,
+                    }
+                }
+            }, (message) => {
+                client2Messages.push(message)
+            }),
+            client3.resend({
+                stream: freshStreamId,
+                resend: {
+                    from: {
+                        timestamp: timeAfterFirstMessagePublished,
+                    }
+                }
+            }, (message) => {
+                client3Messages.push(message)
+            })
+        ])
+
+        await waitForCondition(() => client2Messages.length === 3 && client3Messages.length === 3)
+        await waitForCondition(() => client1Messages.length === 3)
+
+        expect(client1Messages).toEqual([
+            {
+                key: 2
+            },
+            {
+                key: 3
+            },
+            {
+                key: 4
+            },
+        ])
+
+        expect(client2Messages).toEqual([
+            {
+                key: 2
+            },
+            {
+                key: 3
+            },
+            {
+                key: 4
+            },
+        ])
+
+        expect(client3Messages).toEqual([
+            {
+                key: 2
+            },
+            {
+                key: 3
+            },
+            {
+                key: 4
+            },
+        ])
+    })
```

Comment:
Lots of duplication and whitespace in these tests.

At a minimum I'd probably collapse these something like this (or whatever the smallest you can get it and keep the linter happy):
 ```js
 expect(client3Messages).toEqual([
  { key: 2 }, { key: 3 }, { key: 4 },
])
```
Maybe this just matches how other tests in here are set up, but I feel like these should probably be collected into a variable and tested against that, rather than repeated? 

Something like this perhaps, your call though:

```js
// collect all messages to be published so we can check against them later
// could do something like your `for (let i = 1; i <= 3; ++i) {` above 
// or get fancy like `Array(4).fill().map((_, key) => ({ key }))`
// but it's not much shorter than just writing it out (once)
const messages = [{ key: 0 }, { key: 1 }, { key: 2 }, { key: 3 }] // start from 0 so indexes match keys

await client1.publish(freshStreamId, messages[0]))
await wait(5) // doesn't need to be long, wait(1) would also probably suffice?
const timeAfterFirstMessagePublished = Date.now()
await client1.publish(freshStreamId, messages[1]))
await wait(5)
await client1.publish(freshStreamId, messages[2])

await Promise.all([
            client1.resend({
                stream: freshStreamId,
                resend: {
                    from: {
                        timestamp: timeAfterFirstMessagePublished
                    }
                }
            }, (message) => {
                client1Messages.push(message)
            }),
            // etc …
])

// do these all together
await Promise.all([
    // messages.length - 1, should skip first message
    waitForCondition(() => client1Messages.length === messages.length - 1),
    waitForCondition(() => client2Messages.length === messages.length - 1),
    waitForCondition(() => client3Messages.length === messages.length - 1),
])

// compare directly against messages we published 
expect(client1Messages).toEqual(messages.slice(1)) // all but first
// etc …